test-site/scripts/build.sh

```diff
@@ -7,4 +7,6 @@ set_working_dir_to_test_site () {
 
 set_working_dir_to_test_site
 
-npx jambo build && npx grunt webpack
+npx jambo build
+chmod u+x ./public/static/node_modules/esbuild/bin/esbuild
```

Comment:
I don't think we need this step for the script. ideally, there shouldn't be a node_modules folder present in the static dir